my-first-app/src/Components/PostCard/PostCard.tsx

```diff
@@ -0,0 +1,93 @@
+import Like from '../../assets/Icon-Thumbs-Up.svg?react'
+import DisLike from '../../assets/Icon-Thumbs-Down.svg?react'
+import Save from '../../assets/Icon-Bookmark.svg?react'
+
+import style from './PostCard.module.scss'
```

Comment:
style is object and if you have a lot classes which are used in your component you can destructure it 
as example
  const { infoWrap, cardWrap, date } = styles;